templates/csharp/command-and-response/teamsapp.local.yml.tpl

```diff
@@ -67,4 +67,20 @@ provision:
   - uses: teamsApp/update
     with:
       # Relative path to this file. This is the path for built zip file.
-      appPackagePath: ./appPackage/build/appPackage.${{TEAMSFX_ENV}}.zip
\ No newline at end of file
+      appPackagePath: ./appPackage/build/appPackage.${{TEAMSFX_ENV}}.zip
+
+  # Create or update debug profile in lauchsettings file
+  - uses: file/createOrUpdateJsonFile
+    with:
+      target: ./Properties/launchSettings.json
+      content:
+        profiles:
+          Microsoft Teams (browser):
+            commandName: "Project"
+            dotnetRunMessages: true
+            launchBrowser: true
+            launchUrl: "https://teams.microsoft.com/l/app/${{TEAMS_APP_ID}}?installAppPackage=true&webjoin=true&appTenantId=${{TEAMS_APP_TENANT_ID}}&login_hint=${{USER_NAME}}"
```

Comment:
Is `USER_NAME` a built-in variable or output to user's env file?

Reply:
It will be added to the user's local env file in the prepare teams app stage.

Reply:
How about using a more specific name e.g. `TEAMSFX_M365_USER_NAME`?